core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/fields/content-type-fields-row/content-type-fields-row.component.scss

```diff
@@ -26,6 +26,8 @@
     margin: 0 $spacing-1;
     padding-bottom: 0;
     position: relative;
+    // we need to make the width: 0 to give every item the same start point to grow.
+    width: 0;
```

Comment:
Those are flex items, we need to use flex-shitk: 0 so none can shirnk.

Reply:
I tried but it's not working.
![image](https://user-images.githubusercontent.com/113915849/210820449-33bcc49c-03e9-47b8-a49a-3a4da442e5db.png)